homeassistant/components/konnected/__init__.py

```diff
@@ -355,6 +352,11 @@ async def update_sensor(self, request: Request, device_id) -> Response:
                 "unregistered device", status_code=HTTP_BAD_REQUEST
             )
 
+        panel = device.get("panel")
+        if panel is not None:
+            # connect if we haven't already
+            hass.async_create_task(panel.async_connect())
```

Comment:
We don't want to wait for the connect to finish?

Reply:
We don't need to wait as the response of this method doesn't depend on the connect operation. - we just want to trigger a connect attempt ASAP.